docker-compose-dev.yml

```diff
@@ -22,6 +22,7 @@ services:
         - "web:127.0.0.1"
     ports:
       - "8000:8000"
+#      - "7887:7887"
```

Comment:
is there any reason to have this comment?

Reply:
That port needs to be exposed for the notifications to work under docker.
I think this can be left here if the purpose is clarified.

Reply:
Added comments to clarify. I decided to leave port 7887 commented as notifications server still requires additional, manual setup (downloading node.js etc.)